tests/run_tests.sh

```diff
@@ -1,5 +1,27 @@
 #!/bin/bash
 
+# Helper function to print debugging data for failures on graphical environment.
+function print_graphics_data () {
+#	echo "# ***********************************************"
```

Comment:
Let's include the requirement for the env var to print this stuff - the CI passes it, and then it's opt-in for users running this manually.
```suggestion
function print_graphics_data () {
	if [[ -z ${PRINT_GLXINFO} ]]; then
		return;
	fi
#	echo "# ***********************************************"
```